index-en.md

```diff
@@ -12,8 +12,8 @@ lang: en
 		<p>The page templates and design patterns below comprise a reference implementation of the <a href="https://design.canada.ca">Canada.ca design system</a>, including the mandatory requirement of the Content and Information Architecture (C&amp;IA) Specification. Government of Canada departments and agencies can contribute additional patterns and templates via <a href="https://github.com/wet-boew/GCWeb">GCWeb github repository</a>.</p>
 	</div>
 	<div class="col-xs-12 col-md-auto pull-right">
-		<p><a href="https://github.com/wet-boew/GCWeb/archive/v9.5.0.zip" class="btn btn-primary">Download GCWeb theme <strong>v9.5.0</strong></a><br />
-			<small>(<time>{{ page.dateModified | %F }}</time> - <a href="https://github.com/wet-boew/gcweb/releases/tag/v9.5.0">Release notes</a>)</small></p>
+		<p><a href="https://github.com/wet-boew/GCWeb/archive/v10.0.0.zip" class="btn btn-primary">Download GCWeb theme <strong>v10.0.0</strong></a><br />
```

Comment:
```suggestion
		<p><a href="https://github.com/wet-boew/GCWeb/archive/v10.0.0.zip" class="btn btn-primary">Download GCWeb theme <strong>v10.0.0 (STR)</strong></a><br />
```

Reply:
We didn't added STR in the index HTML page, but only in the release note